account_reconcile_pos_payments/models/account_bank_statement.py

```diff
@@ -210,28 +214,26 @@ def process_statement_reconcile(self, lines, statement_id, ct_line=False):
                 'credit': abs(line.amount),
                 'journal_id': self.journal_id.id,
                 'date': statement_id.date,
-                'account_id': statement_id.journal_id.default_credit_account_id.id,
+                'account_id':
+                    statement_id.journal_id.default_credit_account_id.id,
             }
 
             _logger.debug('Creating reconciliation line: %s' % move_line_vals)
             line.process_reconciliation([], [], [move_line_vals])
             # Now let's settle this line with the statement lines
-            lines_to_reconcile = []
 
             for move_line in statement_id.move_line_ids:
-                if (
-                        move_line.account_id.id == statement_id.journal_id.default_debit_account_id.id
-                        and move_line.id not in lines_to_reconcile
-                ):
+                if (move_line.account_id.id ==
+                        statement_id.journal_id.default_debit_account_id.id
+                        and move_line.id not in lines_to_reconcile):
                     lines_to_reconcile.append(move_line.id)
 
             for move_line in line.journal_entry_ids:
-                if (
-                        move_line.account_id.id == statement_id.journal_id.default_credit_account_id.id
-                        and move_line.id not in lines_to_reconcile
-                ):
+                if (move_line.account_id.id ==
+                        statement_id.journal_id.default_credit_account_id.id
+                        and move_line.id not in lines_to_reconcile):
                     lines_to_reconcile.append(move_line.id)
 
-            move_lines = self.env['account.move.line'].browse(
-                lines_to_reconcile)
-            move_lines.reconcile()
+        move_lines = self.env['account.move.line'].browse(
+            lines_to_reconcile)
+        move_lines.reconcile()
```

Comment:
So, the only actual change is in the indentation of these 3 lines, right?

Reply:
@ivantodorovich .. yes